build_requirements/dev/pyproject.toml

```
@@ -0,0 +1,65 @@
+[tool.poetry]
+name = "qml"
+version = "0.0.0-dev"
+description = "Introductions to key concepts in quantum machine learning, as well as tutorials and implementations from cutting-edge QML research."
+authors = ["PennyLaneAI <info@pennylane.ai>"]
+license = "Apache-2.0"
+readme = "README.md"
```

unless it was skipped on purpose
```suggestion
readme = "README.md"
homepage = "https://pennylane.ai/qml/"
```

---

Ah whoops, I'll add that back